ml/src/main/scala/org/apache/spark/ml/h2o/models/H2OMOJOReader.scala

```diff
@@ -18,14 +18,37 @@
 package org.apache.spark.ml.h2o.models
 
 import org.apache.hadoop.fs.Path
+import org.apache.spark.ml.h2o.SparkSpecificUtils
 import org.apache.spark.ml.param.Params
+import org.apache.spark.ml.util.DefaultParamsReader.Metadata
 import org.apache.spark.ml.util._
 import org.apache.spark.sql._
 import org.apache.spark.util.Utils
-import org.json4s.JsonAST
+import org.json4s.JsonAST.JObject
+import org.json4s.jackson.JsonMethods.{compact, render}
+import org.json4s.{DefaultFormats, JsonAST}
 
 private[models] class H2OMOJOReader[T <: HasMojoData] extends DefaultParamsReader[T] {
 
+  private def getAndSetParams(
+                               instance: Params,
+                               metadata: Metadata,
+                               skipParams: Option[List[String]] = None): Unit = {
```

Comment:
nit: Do we need an option here? Wouldn't `Nil` play the same role as `None`?

Reply:
DTTO